nixos/modules/services/web-apps/nextcloud-notify_push.nix

```diff
@@ -41,14 +41,12 @@ in
       "dbtype"
       "dbname"
       "dbuser"
-      "dbpassFile"
       "dbhost"
-      "dbport"
       "dbtableprefix"
     ] (
       opt: options.services.nextcloud.config.${opt} // {
```

Comment:
AFAIU this only works because of `mkRenamedOptionModule`, correct?
This seems wrong to me because removing `mkRenamedOptionModule` and friends shouldn't need additional fixes.

Haven't checked if that's sufficient by switching this to `settings`, though I doubt it. If not, I'm wondering if we should just duplicate these options and use as description something like "same as nextcloud option" (with a proper link and better wording though).